pom.xml

```
@@ -88,6 +88,11 @@
             <artifactId>jackson-databind</artifactId>
             <version>2.12.1</version>
         </dependency>
+        <dependency>
+        	<groupId>javax.annotation</groupId>
+        	<artifactId>javax.annotation-api</artifactId>
+        	<version>1.3.2</version>
+        </dependency>
```

Давай всюди використовувати однакове форматування
```suggestion
        <dependency>
            <groupId>javax.annotation</groupId>
            <artifactId>javax.annotation-api</artifactId>
            <version>1.3.2</version>
        </dependency>
```